assets/scripts/menus/ShareMenu.scss

```diff
@@ -11,14 +11,20 @@ $share-menu-width: 350px;
   [dir="rtl"] & {
     padding: 0.5em 2.5em 0.5em 1em;
   }
+
+  button {
+    border: 1px solid #a9ccdb;
```

Comment:
Let's actually remove the full border. The border style might change in the future since just having a top and bottom border is uncommon, but it's the style we have in most situations in Streetmix right now.